### PR TITLE
Add styles for desktop view

### PR DESCRIPTION
**Pull request by MesakDuduCoder**
Added styles for desktop and updated index.html layout for improved readability.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,81 +25,29 @@
...
+            <nav class="nav-bar">
+                <div class="nav-bar-frame"><a href="/portfolio/">Portfolio</a></div>
+                <div class="nav-bar-frame"><a href="/about/">About</a></div>
+                <div class="nav-bar-frame"><a href="/contact/">Contact</a></div>
+                <button id="header-btn2">
+                    <nav>
+                      <img src="./icons/Union.png" alt="">
+                    </nav>
+                </button>
+            </nav>
```
Comment (jerryowusu) [inline index.html:37]: - [ ] The navigation menus are missing from your desktop design. Kindly check and make sure they are present to match the actual design. 
- [ ] The headline part of your design seems to be pushed towards the left side with the background covering that part alone leaving a blank empty space to the right as shown below. Kindly check and fix it to match the design. 

### Your design 

Screenshot from 2023-03-03 19-17-21



### Figma design


Screenshot from 2023-03-03 19-19-15

File: desktop.css [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+    width: 100%;
+    height: 100%;
+    padding-left: 8%;
+    padding-right: 8%;
+    padding-bottom: 20%;
+    box-sizing: border-box;
+    border-top-right-radius: 150px;
+    margin-top: -12%;
+    z-index: 1;
+  }
```
Comment (jerryowusu) [inline desktop.css:270]: This section contains 2 blocks. The left one takes 40% and the right one take the remaining 60% of the page's width. However, I see that your design has partitioned them into two, each occupying 50%. Kindly check and correct this anomaly in order to match the actual design. 

### Your design 
Screenshot from 2023-03-03 19-23-07

### Actual design

Screenshot from 2023-03-03 19-23-25

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @MesakDuduCoder , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

no-we-were-so-close-pauline-turner-brooks

## Highlights
No linter errors :heavy_check_mark: 
Used github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -269,51 +215,35 @@ <h2 class="project-title">Tonic</h2>
...
-                Myself
-            </h3>
-            <p>
-                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
-                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            <div class="social-media">
-                <div class="socials-text">
+            <div class="social-text">
+                <h3>About Myself</h3>
```
Comment (tufoinnkuo10) [inline index.html:228]: - [optional] kindly reduce the extra white space above the about heading and also please follow the about myself text design. 
yours
Screenshot (850)_LI

figma

Screenshot (851)_LI

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MesakDuduCoder 

Your project is complete! Great job making the changes. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design on point
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**